dotnet/samples/Demos/ContentSafety/Controllers/ChatController.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using ContentSafety.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.SemanticKernel;
+
+namespace ContentSafety.Controllers;
+
+/// <summary>
+/// Sample chat controller.
+/// </summary>
+[ApiController]
+[Route("[controller]")]
+public class ChatController(Kernel kernel) : ControllerBase
+{
+    private readonly Kernel _kernel = kernel;
+
+    [HttpPost]
+    public async Task<IActionResult> PostAsync(ChatModel chat)
+    {
+        this._kernel.Data["documents"] = chat.Documents;
+
+        return this.Ok((await this._kernel.InvokePromptAsync(chat.Message)).ToString());
```

Comment:
Might be good to double check with @markwallace-microsoft about this pattern, where we pass the user message to the kernel without using a template, since it allows the user to inject system prompts, and we are trying to avoid recommending it to folks.

Reply:
Makes sense, I've added prompt template with kernel arguments, which should be encoded by default.